extension/scripts/features/jailfilter/ttJailFilter.js

```diff
@@ -0,0 +1,246 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Jail Filter",
+		"jail",
+		() => settings.pages.jail.filter,
+		null,
+		addFilters,
+		removeFilters,
+		{
+			storage: ["settings.pages.jail.filter"],
+		},
+		async () => {
+			await requireElement(".users-list > *:first-child .info-wrap");
+		}
+	);
+
+	function addFilters() {
+		const { content } = createContainer("Jail Filter", {
+			nextElement: document.find(".users-list-title"),
+		});
+		content.innerHTML = `
+			<div class="filter-header">
+				<div class="statistic" id="showing">Showing <span class="filter-count">X</span> of <span class="filter-total">${document.findAll(".users-list > li").length}</span> users</div>
+			</div>
+			<div class="filter-content">
+				<div class="filter-wrap" id="activity-filter">
+					<div class="filter-heading">Activity</div>
+					<div class="filter-multi-wrap">
+						<div class="tt-checkbox-wrap">
+							<input type="checkbox" value="online" id="tt-hospital-filter-online">
+							<label for="tt-hospital-filter-online">Online</label>
+						</div>
+						<div class="tt-checkbox-wrap">
+							<input type="checkbox" value="idle" id="tt-hospital-filter-idle">
+							<label for="tt-hospital-filter-idle">Idle</label>
+						</div>
+						<div class="tt-checkbox-wrap">
+							<input type="checkbox" value="offline" id="tt-hospital-filter-offline">
+							<label for="tt-hospital-filter-offline">Offline</label>
+						</div>
+					</div>
+				</div>
+				<div class="filter-wrap">
+					<div class="filter-heading">Faction</div>
+					<select name="faction" id="tt-faction-filter">
+						<option selected value="">none</option>
+						<option disabled value="------">------</option>
+						<option value="${userdata.faction.faction_tag}">${userdata.faction.faction_tag}</option>
+					</select>
+				</div>
+			</div>
+		`;
+		const timeFilter = document.newElement({
+			type: "div",
+			class: "filter-wrap",
+			id: "time-filter",
+			children: [
+				newSlider(),
+				document.newElement({
+					type: "div",
+					class: "time-filter-info-wrap",
+					children: [document.newElement({ type: "span", class: "time-filter-info" })],
+				}),
+			],
+		});
+		const levelFilter = document.newElement({
+			type: "div",
+			class: "filter-wrap",
+			id: "level-filter",
+			children: [
+				newSlider(),
+				document.newElement({
+					type: "div",
+					class: "level-filter-info-wrap",
+					children: [document.newElement({ type: "span", class: "level-filter-info" })],
+				}),
+			],
+		});
+		const scoreMax =
+			Math.max(100 * (parseInt(document.find(".users-list > *:first-child .info-wrap .time").lastChild.textContent.trim().split(" ")[0].replace(/[hs]/g, "")) + 3), 5000);
```

Comment:
Score should be in a function, should it ever change we'll only have to edit it in 1 place.